gems/sorbet-runtime/lib/types/private/types/not_typed.rb

```diff
@@ -12,7 +12,7 @@ def name
   end
 
   # @override Base
-  def valid?(obj)
+  def valid?(obj, deep=false)
```

Comment:
Why is this positional while the others are keyword? I generally prefer that optional boolean arguments are keyword arguments.

Reply:
On second thought, seeing how this is used throughout the typed_enumerable code below, it seems really easy to forget to forward through the `deep` parameter. Can we make this parameter required, and change pay-server?